stories/alert/props-table.tsx

```diff
@@ -70,5 +70,20 @@ export const PropsTable = () => (
                 </>
             </DefaultCol>
         </tr>
+        <tr>
```

Comment:
Could you refactor this to use the object structure? Can refer to the other props table like `Button`